### DISCUSSION TITLE
Work with Ocelot Dynamic route

### DISCUSSION THREAD

**Discussion by slarkerino** [Burgyn/MMLib.SwaggerForOcelot#154] [Ideas]
Can this library integrate with ocelot's dynamic route feature?

**Comment by Burgyn** [comment]
Hi @slarkerino,

thanks for sharing your idea. 

Your idea is correct. It would be nice if this package supported Dynamic route. At the moment, unfortunately, I do not plan to implement this _(because of priorities 🙄)_.

This package does not directly use repositories from the Ocelot to obtain routing, so redesign would be a bit more complicated.

If there is a lot of interest in it _(more people need it)_, I can look at it. Alternatively, if anyone want to implement it, write me and I will try to direct you.
